### ISSUE TITLE
[Bug] service account authentication failure for crawling urls

### ISSUE THREAD

**Issue by v-viyada** [microsoft/accessibility-insights-service#2569]
## Describe the bug

accessibility insights scan is failing to scan url using service account authentication and giving error message  Authentication failed. Authentication requires a non-people service account. To learn how to set up a service account, visit: https://aka.ms/AI-action-auth. TargetCloseError: Waiting for selector `#FormsAuthentication` 

This is used to run fine earlier with same service account and there is no change in service account password as well.

## Area

Crawler package and may be scanner global library

## To Reproduce

1. Build cli package
2. Run below command
node ./node_modules/accessibility-insights-scan/dist/ai-scan-cli.js --url <url> --authType AAD --serviceAccountName <name of service account> --serviceAccountPassword <service account password> --restart --silentMode false --crawl true --maxUrls 1

## Expected behavior

Service account authentication should succeed and scan the URL

## Additional context

Investigate the issue and after inputting service account and hit enter, the code waits for selector `#FormsAuthentication`. If the selector does not load it gives the error mentioned in bug description. 
I observed that login page to enter password does not have `#FormsAutentication` selector, please refer below screenshot from inspect element
![image](https://github.com/user-attachments/assets/deee9363-9e4c-4ff1-9ebe-a1173f6ccb1f)
